stripe/src/main/java/com/stripe/android/model/Card.java

```diff
@@ -280,6 +280,8 @@ public String getNumber() {
 
     public void setNumber(String number) {
         this.number = number;
+        this.type = "";
+        this.type = getType();
```

Comment:
This is pretty minor, but I think that `getType()` will always return a value so you can probably just leave out blanking `this.type` on the previous line completely.